Feature/ch61b

RaulRory opened the pull request:
- Atualizando a a Branch conforme os outros PR's
- adicionando verificação de usuarios qunado for pegar o token
- Usando linter do Standart js para deixar o code style padrão

OBS: Mergear e ATualizar os PR's

Commit:
Config padrão Standart para code Style

Commit:
Atualizando a branch

Commit:
Atualizando branch com verificação de usuario

mbj-junior commented on package.json:
```diff
@@ -7,7 +7,8 @@
     "start:dev": "nodemon ./src/bin/server.js",
     "start:db": "node ./src/bin/syncDB.js",
-    "test": "jest ./test/app.test.js"
+    "test": "jest ./test/app.test.js",
```
Esse campo irá conflitar com as alterações da branch feature/ch62

mbj-junior commented on src/controllers/log.js:
```diff
@@ -1,59 +1,47 @@
 
-Log.save = async (req, res, next) => {
-    const result = await req.body
+Log.getAll = async (req, res, next) => {
+  const data = await model.findAll({})
```
Conflito com a feature/ch62

mbj-junior committed:
Delete testeCrypto.js

Arquivo usado para testar as funcionalidades de criptografia e descriptografia.

Commit:
atualizando função para gerar o token

Commit:
Merge branch 'feature/ch61b' of https://github.com/codenation-dev/squad-4-ad-nodejs-kroton-1 into feature/ch61b

RaulRory commented:
Branchs autualizadas e PR's Aprovados
